packages/augur-ui/src/modules/market-cards/containers/market-card.ts

```diff
@@ -34,6 +34,14 @@ const mapDispatchToProps = (dispatch, ownProps) => ({
         selectedOutcome,
       })
     ),
+  migrateMarketModal: () =>
+    dispatch(
+      updateModal({
+        type: MODAL_MIGRATE_MARKET,
+        market: ownProps.market,
+        gasCost: MIGRATE_MARKET_GAS_ESTIMATE,
```

Comment:
why are you passing this in here

Reply:
Good point, this doesn't need to be passed in. I'll remove it.